### PR TITLE
Update doc for CA1304 to mention excluded symbols

### LINKED ISSUE #4992: CA1304 - configurability section - excluded symbol names
With the release of fxcop 3.x, the rule provide a user-option to let users define symbols excluded by this rule.

You can copy the doc from https://docs.microsoft.com/fr-fr/visualstudio/code-quality/ca3006?view=vs-2019#excluded-symbol-names

See https://github.com/dotnet/roslyn-analyzers/pull/3325

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 909fe7fe-25bb-7923-a6a0-15b7fca68dfa
* Version Independent ID: 0650243f-aea9-149d-e6ec-65e0dedcbd78
* Content: [CA1304: Specify CultureInfo - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1304?view=vs-2019)
* Content Source: [docs/code-quality/ca1304.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/code-quality/ca1304.md)
* Product: **visual-studio-windows**
* Technology: **vs-ide-code-analysis**
* GitHub Login: @Mikejo5000
* Microsoft Alias: **mikejo**

### PR DESCRIPTION
**Pull request by Evangelink**
Fix #4992

This new feature is not yet publicly released. Please sync-up with @mavasani for when to merge this PR.

### CODE REVIEW

**Comment by PRMerger13** [comment]
@Evangelink : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/code-quality/ca1304.md [hunk 1]
```diff
@@ -74,6 +74,35 @@ This example produces the following output:
 06/04/1900 12:15:12
 ```
 
+## Configurability
```
Comment (mavasani) [inline docs/code-quality/ca1304.md:77]: Please move the section above the examples and also add a small sentence about configurability at the end of `Cause` section, with a hyperlink to here.

**Comment by ktoliver** [comment]
> LGTM, modulo the suggestions.

@mavasani I'm not sure what this means. Is the PR ready to merge or are revisions required before it's approved for merge? Thanks.

**Comment by mavasani** [comment]
@ktoliver - sorry, I meant once the suggested changes are made, we should be good to merge. Doesn't really need a second review pass...

**Comment by ktoliver** [comment]
@mavasani - Got it. Thanks.

**Comment by ktoliver** [comment]
Hi @Evangelink - Could you make the revisions that @mavasani requests? Then we can merge the PR. Thanks.

**Comment by Mikejo5000** [comment]
#sign-off

**Pull request merged**